src/js/helpers/handleScroll.js

```diff
@@ -24,9 +24,10 @@ const setState = function (el, top, once) {
  * @param  {bool} once               plugin option
  * @return {void}
  */
-const handleScroll = function ($elements, once) {
-  const scrollTop = window.pageYOffset;
-  const windowHeight = window.innerHeight;
+const handleScroll = function ($elements, once, container) {
```

Comment:
Please update the documentation aboe this method and add `container`. (note: duplicate of #393)